sycl/test-e2e/AddressSanitizer/lit.local.cfg

```diff
@@ -7,3 +7,7 @@ config.substitutions.append(
 config.substitutions.append(
     ("%force_device_asan_rt", "env SYCL_PREFER_UR=1 UR_ENABLE_LAYERS=UR_LAYER_ASAN")
 )
+
+# DeviceSanitizer only supports CPU, DG2 and PVC
+if not ('gpu-intel-dg2' in config.available_features or 'gpu-intel-pvc' in config.available_features):
+  config.required_features += ['cpu']
```

Comment:
So, if we run on the systems with these cards only, then the test will never be executed on CPU, I think. Doesn't look right to me.

Reply:
No, it means if the machine has PVC or DG2 it will run on both CPU and GPU, otherwise it runs on CPU only.

Reply:
I changed the algorithm and now it can support all intel gpu device.